docs/index.yaml

```diff
@@ -21,11 +21,11 @@ entries:
     name: splunk-enterprise
     type: application
     urls:
-    - https://splunk.github.io/splunk-operator/splunk-enterprise-2.3.0.tgz
+    - https://vivekr-splunk.github.io/splunk-operator/splunk-enterprise-2.3.0.tgz
```

Comment:
this is not correct, please correct it to point to splunk.github.io  and same for other places as well

Reply:
fixed